acl/src/main/java/org/apache/rocketmq/acl/plain/PlainAccessResource.java

```diff
@@ -281,6 +380,43 @@ public static PlainAccessResource build(PlainAccessConfig plainAccessConfig, Rem
         Permission.parseResourcePerms(plainAccessResource, false, plainAccessConfig.getGroupPerms());
         Permission.parseResourcePerms(plainAccessResource, true, plainAccessConfig.getTopicPerms());
 
+        List<ResourceAndPerm> resourcePerms = plainAccessConfig.getResourcePerms();
```

Comment:
It may be better to wrap the process of converting new resource and namespace perm in `Permission`.